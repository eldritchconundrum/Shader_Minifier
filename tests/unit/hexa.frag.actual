/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef HEXA_FRAG_ACTUAL_
# define HEXA_FRAG_ACTUAL_

const char *hexa_frag =
 "void main()"
 "{"
   "float f=float(255)/1000.,r=float(-255)/1000.,o=f-r+float(0);"
   "gl_FragColor=vec4(.2,.4,f,0.);"
 "}";

#endif // HEXA_FRAG_ACTUAL_
