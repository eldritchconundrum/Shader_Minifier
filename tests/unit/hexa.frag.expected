/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef HEXA_FRAG_EXPECTED_
# define HEXA_FRAG_EXPECTED_

const char *hexa_frag =
 "void main()"
 "{"
   "float f=float(255)/1000.,r=float(-255)/1000.,o=f-r+float(0);"
   "gl_FragColor=vec4(.2,.4,f,0.);"
 "}";

#endif // HEXA_FRAG_EXPECTED_
